### DISCUSSION TITLE
Add a new feature flag for media players which support announcements

### DISCUSSION THREAD

**Discussion by emontnemery** [home-assistant/architecture#941] [Entity Models] [answered]
### Suggestion:
Add a new feature flag `MediaPlayerEntityFeature.ANNOUNCE` for media players which support announcements

### Rationale:
Support for announcements when calling the `media_player.play_media` is not mandatory, and only supported by some integrations. By adding a feature flag, it's clear if the media players supports announcements

**Comment by frenck** [comment, marked as answer]
Do it! 👍 

../Frenck

**Reply by frenck** [reply]
Implemented in https://github.com/home-assistant/core/pull/95906
